### PR TITLE
Update 03-prepare-user-data.md

### PR DESCRIPTION
**Pull request by rishi-anand**
## Describe the Change

This PR updates User Data examples for edge-native clusters.

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
Welcome @rishi-anand! It looks like this is your first PR to spectrocloud/librarium 🎉

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:288]: I don't see a change here.

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (ritawatson) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:161]:
```suggestion
**Site** - supplied at the edge location through a bootable USB drive. If specified, the `projectName` value overrides project information specified in the user data's edge host registration token. Optionally, you can add tags `tags:city`, `tags:building`, and `tags:zip-code`.
```

**Review by ritawatson — APPROVED** [review]
Minor comments. @karl-cardenas-coding you might want to double-check my suggested changes. Otherwise, LGTM.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1261#" title="Author self-approved">rishi-anand</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1261#pullrequestreview-1402860135" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:126]:
```suggestion
# There is no password specified to the default kairos user. You must specify authorized keys or passwords to access the Edge host console. 
```

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:90]:
```suggestion
    # Optional
    # If the Edge host requires a proxy to connect to Palette or to pull images, then specify the proxy information in this section
```

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:86]:
```suggestion
    # The device's name, may also be referred to as the Edge ID or Edge host ID.  If no Edge hostname is specified,
    # a hostname will be generated from the device serial number.  If the Edge Installer cannot identify the device serial number, then a random ID will
    # be generated and used instead. In the case of hardware that does not have a serial number, we recommended specifying a
    # random name, with minimal chances of being re-used by a different Edge host.
```
Language cleanup.

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:76]:
```suggestion
    # Tags which that will be assigned to the device as labels.
```

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:74]:
```suggestion
    # The Palette project ID the Edge host should pair with. This is an optional field if an edgeHostToken is used and the token was assigned to a project.
```

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:66]:
```suggestion
    # The Palette API endpoint to use. The default value is api.spectrocloud.com.
```

File: content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/03-edge/04-edgeforge-workflow/03-prepare-user-data.md:71]:
```suggestion
    # The edgeHostToken is an auto-registration token to register the Edge host with Palette upon bootup.
    # This token can be generated by navigating to the Tenant Settings -> Registration Token.
    # Specify a name and select the project id with which the Edge host should register itself.
```

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good @rishi-anand just some minor language cleanup suggestions.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1261#pullrequestreview-1402864498" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1261#" title="Author self-approved">rishi-anand</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1261#pullrequestreview-1402860135" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
